examples/ibm-catalog/deployable-architectures/extension/README.md

```diff
@@ -1,66 +1,65 @@
 # IBM Cloud catalog example for Power infrastructure for regulated industries
 
-This example illustrates how to use the `power-infrastructure` module in the IBM catalog. It provisions the following infrastructure:
+This example extends PowerVS infrastructure for deployable architectures in variation PowerVS Workspace incl. Secure Landing Zone (Full Stack solution) with an additional PowerVS Workspace.
```

Comment:
This example extends an existing PowerVS infrastructure for deployable architectures deployed as full-stack with an additional PowerVS workspace.

Reply:
Done